src/base/bittorrent/torrentimpl.cpp

```diff
@@ -96,6 +96,27 @@ namespace
             entryList.emplace_back(setValue.toStdString());
         return entryList;
     }
+
+    void initializeStatus(lt::torrent_status &status, const lt::add_torrent_params &params)
```

Comment:
Maybe name it `fillStatus()`? It doesn't feel like it is initializing something.

Reply:
>It doesn't feel like it is initializing something.

It initializes the Status with the initial values for the first use. Unlike other functions, it is OK to name structural functions (i.e. those that simply serve to divide one large function into smaller parts and are used only in a single place) by their use.

Reply:
Or you could make it a local lambda so its name won't matter.

Reply:
>Or you could make it a local lambda

The point of its existence is to avoid making the TorrentImpl constructor too large, by dividing it into structural components. What lambda can we talk about?

Reply:
well forget it.